### DISCUSSION TITLE
Is there a way to access clicked row values in table block?

### DISCUSSION THREAD

**Discussion by xlog-null** [Budibase/budibase#8130] [Help] [answered]
When you link a table row to a url when clicked, you could set the url to e.g. /screen/:id with id being the of the row. But I want to do a conditional based on a field in the row, e.g. if on field of row is type1, link to screen /type1_details/:id and if it's type2, link to /type2_details/:id.

Is this possible?

**Comment by andz-bb** [comment, marked as answer]
Hey @xlog-null, 

This should be possible for you using the Handlebars/JS bindings for the link URL. If you have screens for `/type1_details/:id` and `/type2_details/:id` and a `type_` field in your table you could use a Handlebars binding like this: `/{{ Table.Stuff.type_ }}/{{ Table.Stuff._id }}`. If you need to use more complex logic than that I would recommend returning the URL string using a JS binding. If you need a hand with that let me know and I'll be happy to help 👍
